lib/stateTransition/fetchLockTransactionOutputFactory.js

```diff
@@ -43,7 +46,17 @@ function getLockedTransactionOutputFactory(
       throw new IdentityLockTransactionNotFoundError(transactionHash);
     }
 
-    const transaction = new Transaction(rawTransaction);
+    if (enableLockTxOneBlockConfirmationFallback) {
```

Comment:
This fallback means if we don't have locks we should check confirmations as well.